contrib/src/main/java/com/datatorrent/contrib/avro/AvroRecordHelper.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.datatorrent.contrib.avro;
+
+import java.text.ParseException;
+import java.util.List;
+
+import org.apache.avro.Schema;
+import org.apache.avro.Schema.Type;
+
+/**
+ * This is an utility class for Avro converted records
+ */
+
+public class AvroRecordHelper
+{
+
+  /**
+   * Convert a passed String value to the given type for the key as per Schema
+   * 
+   * @param schema
+   * @param key
+   * @param value
+   * @return
+   */
+
```

Comment:
Please remove spaces between start of function line and comment. The comment may not get picked up for javadocs.